ledger-tool/src/main.rs

```diff
@@ -564,15 +567,28 @@ fn load_bank_forks(
             compression: CompressionType::Bzip2,
         })
     };
+    let blockstore = open_blockstore(&ledger_path, access_type);
     let account_paths = if let Some(account_paths) = arg_matches.value_of("account_paths") {
+        if !blockstore.is_primary_access() {
+            // Be defenstive, when default account dir is explicitly specified, it's still possible
+            // to wipe the dir possibly shared by the running validator!
+            panic!("custom accounts path is not supported under secondary access");
```

Comment:
I'd rather we not panic! in response to user input:
```suggestion
            eprintln!("Error: custom accounts path is not supported under secondary access");
            process::exit(1)
```